tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1382,6 +1382,18 @@ pub trait Filter<S> {
         None
     }
 
+    /// Called before the filtered subscribers' [`on_event`], to determine if
+    /// `on_event` should be called.
+    ///
+    /// This gives a chance to filter events based on their fields.
+    ///
+    /// [`on_event`]: crate::subscribe::Subscribe::on_event
+    #[inline] // collapse this to a constant please mrs optimizer
```

Comment:
i like how the optimizer is, apparently, married :)

Reply:
[:)](https://twitter.com/bitshiftmask/status/1552063086315593729)